src/tools/m65.c

```diff
@@ -280,7 +282,9 @@ int virtual_f011_read(int device, int track, int sector, int side)
   }
 
   /* signal done/result */
```

Comment:
So weit ich sehen kann, liegt die wichtigen Änderung hier und bei Zeilen 343 bis 345, oder?
D.H., dass den CPU muss am Halten gebracht werden, bevor $FFD3086 aktualisiert darf, wenn ich es richtig verstanden haben.

Reply:
Correct, otherwise there is a chance to miss the next VF011 read request from the HYPPO.